models/experimental/functional_unet/tt/unet_shallow_ttnn.py

```diff
@@ -164,23 +169,23 @@ def __init__(
         self.conv_config = ttnn.Conv2dConfig(
             dtype=activation_dtype,
             weights_dtype=weights_dtype,
-            math_fidelity=ttnn.MathFidelity.LoFi,
             shard_layout=shard_layout,
             deallocate_activation=self.deallocate_activation,
-            fp32_dest_acc_enabled=True,
-            packer_l1_accum_enabled=False,
             enable_act_double_buffer=(
                 conv.use_activation_double_buffer if "use_activation_double_buffer" in conv else False
             ),
-            enable_split_reader=conv.use_split_reader if "use_split_reader" in conv else False,
             enable_subblock_padding=False,
             activation=activation,
             output_layout=ttnn.TILE_LAYOUT,
             input_channels_alignment=conv.input_channels_alignment if "input_channels_alignment" in conv else 32,
+            act_block_div=act_block_div,
+        )
+        self.conv_compute_config = ttnn.WormholeComputeKernelConfig(
```

Comment:
why didn't you use GetComputeKernelConfig here?

Reply:
Fixed.